DiscordChatExporter.Core/Exporting/Filtering/FromMessageFilter.cs

```diff
@@ -0,0 +1,19 @@
+﻿using DiscordChatExporter.Core.Discord.Data;
+
+namespace DiscordChatExporter.Core.Exporting.Filtering
+{
+    public class FromMessageFilter : MessageFilter
+    {
+        private readonly string _value;
+
+        public FromMessageFilter(string value)
+        {
+            ///TODO: should we validate input here?
+            _value = value;
+        }
```

Comment:
I decided against rejecting input that fails to look like a username+discriminator or a snowflake (right now it'll just fail to match anything), but I thought it might be better to throw an exception before starting the export.

Reply:
I think this is fine for now